docs/conf/cincinnati/2018/cfp.rst

```diff
@@ -5,7 +5,8 @@ Call for Proposals
 
 Hello hello, fellow documentarians!
 
-It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next {{ area_adj }} conference, coming up on {{date.main}}.
+It's that time of year again: We’re very excited to announce that we are now accepting talk proposals for our next {{ area_adj }} conference, coming up on {{date.main}}. The conference talk portion of the event will be **August 18**,
+so this is the date you would need to be in attendance. 
```

Comment:
Looks good to me, unless you want to add "...so this is the date you would need to be in attendance if your proposal is accepted." I think it's fine as written though!